zendesk/ticket.go

```diff
@@ -9,33 +9,38 @@ import (
 	"time"
 )
 
+type CustomField struct {
+	ID    int64  `json:"id"`
+	Value string `json:"value"`
```

Comment:
How do we handle the cases where the value is set to an array of values?

See:
https://developer.zendesk.com/rest_api/docs/support/tickets#setting-custom-field-values

Reply:
Yeah, it should be left as `interface{}`. 

Here is an example.

https://github.com/nukosuke/go-zendesk/blob/ad03611ba437c4511e9a85e0cf2fb687d296b62c/zendesk/trigger.go#L22-L25

Reply:
I have started on fixing it to work with `interface{}`.
Do you prefer that I just leave it similar to `TriggerAction`?
I was going by another example you have in the code-base:
https://github.com/nukosuke/go-zendesk/blob/ad03611ba437c4511e9a85e0cf2fb687d296b62c/zendesk/collaborators.go#L15-L20
https://github.com/nukosuke/go-zendesk/blob/ad03611ba437c4511e9a85e0cf2fb687d296b62c/zendesk/collaborators.go#L77-L78
Trying to ensure that the values are the correct type (especially the `int64`).

Reply:
We provide `Collaborators` struct because the payload is too complex to simply handle with interface, which can take [these different types](https://developer.zendesk.com/rest_api/docs/support/tickets#setting-collaborators).

In this case, the candidates are `string` or `[]string`. I think that `interface` is enough to handle.

Reply:
OK, then I'll keep it simple and just leave it as `interface{}`.

Reply:
I changed `CustomField.Value` to `interface{}`.